sdk/core/core-auth/review/core-auth.api.md

```diff
@@ -0,0 +1,39 @@
+## API Report File for "@azure/core-auth"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public
+export interface AbortSignalLike {
```

Comment:
Let's import this from core/abort-controller.

Reply:
Done!